### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 5.13

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: check_process [hunk 1]
```diff
@@ -23,10 +23,8 @@
         setup_private=1
         setup_public=0
         upgrade=1
-        # 5.4~ynh1
-        upgrade=1   from_commit=cfa9c5a3dbcfb765dcca3b2e7d179756afe26def
         # 5.11.4~ynh1
-        upgrade=1   from_commit=3d2f68c4e19f335e63f8ffa259708b38a58c8f67
+        upgrade=1   from_commit=4999654987af8b1427a6c72f8af482b235bb46db
```
Comment (yalh76) [inline check_process:27]: You should keep test from  cfa9c5a3dbcfb765dcca3b2e7d179756afe26def and from 3d2f68c4e19f335e63f8ffa259708b38a58c8f67
and add a test from 4999654987af8b1427a6c72f8af482b235bb46db to be named 5.11.4~ynh2
There was a lot of modifications on those version

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Review by yalh76 — APPROVED** [review]

**Pull request merged**